spec/controllers/hubstats/deploys_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+module Hubstats
+  RSpec.describe DeploysController, :type => :controller do
+    describe "#create" do
+      it 'should create a deploy' do
+        @deploy = build(:deploy)
```

Comment:
I thought these might be going around the problem a little bit. I will play around more.